aclogview/Form1.cs

```diff
@@ -112,9 +113,11 @@ protected override void OnClosing(CancelEventArgs e)
                     newItem.SubItems.Add(record.isSend ? "Send" : "Recv");
                     newItem.SubItems.Add(record.tsSec.ToString());
                     newItem.SubItems.Add(record.packetHeadersStr);
-                    newItem.SubItems.Add(record.packetTypeStr);
+                    newItem.SubItems.Add(record.packetTypeStr);                   
                     newItem.SubItems.Add(record.data.Length.ToString());
                     newItem.SubItems.Add(record.extraInfo);
+                    // This one requires special handling and cannot use function.
+                    newItem.SubItems.Add(record.opcodes[0].ToString("X").Substring(4, 4));

```

Comment:
This throws an exception on header-only records and causes app to crash. Please add a check to make sure record.opcodes.Count > 0, else make this an empty string.